libtcmu.c

```diff
@@ -177,8 +177,8 @@ static int reconfig_device(struct tcmulib_context *ctx, char *dev_name,
 
 	ret = dev->handler->reconfig(dev, &cfg);
 	if (ret < 0) {
-		tcmu_dev_err(dev, "Handler reconfig failed with error %d.\n",
-			     ret);
+		tcmu_dev_dbg(dev, "Handler reconfig failed with error %s.\n",
+		             strerror(-ret));
```

Comment:
There also has some other errors like -EINVAL in glfs.c 
Maybe only when ret == -EOPNOTSUPP, make this message as DEBUG or WARNING, and all the others as what it is.

Reply:
I think it might be ok to just make them all dbg statements, because the caller will log the same info that a reconfigure operation failed and the reason why.

Reply:
For the glfs resize failure case for example, we would want to leave the tcmu_dev_err call in tcmu_glfs_reconfig because it has extra info we can't express with an error code. We do not need to log a message above though since  tcmu_glfs_reconfig logs a detailed error and then the caller (rtslib/targetcli normally) will also log the same info as above.

Reply:
Yeah, if so, that looks good to me.